site/src/middleware.page.ts

```diff
@@ -1,3 +1,5 @@
+/* istanbul ignore file */
+/** @see https://github.com/vercel/next.js/issues/32608#issuecomment-1007439478 about istanbul ignore file */
```

Comment:
I would not introduce `istanbul ignore` for now. Merging is not blocked when Codecov check is failing. Keeping a true picture of what’s covered and what’s not is more beneficial at this stage.

Reply:
Disabled it to test if it'll fix failing integration tests, but no luck. Started a thread on slack about this.